### PR TITLE
[Bug](date) fix regression test test_date_function

### PR DESCRIPTION
**Pull request by Gabriel39**
# Proposed changes

image

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Gabriel39** [comment]
run buildall

**Comment by morningman** [comment]
please open this test case in regression-test.conf

File: be/src/vec/runtime/vdatetime_value.cpp [hunk 1]
```diff
@@ -1795,28 +1795,31 @@ bool DateV2Value<T>::is_invalid(uint32_t year, uint32_t month, uint32_t day, uin
 }
 
 template <typename T>
-void DateV2Value<T>::format_datetime(uint32_t* date_val) const {
```
Comment (yiguolei) [inline be/src/vec/runtime/vdatetime_value.cpp:1798]: should enable this test case in pipeline/conf/p0 and p1

**Comment by Gabriel39** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**